packages/lib/src/components/internal/ClickToPay/components/CtPOneTimePassword/CtPOneTimePasswordInput/CtPOneTimePasswordInput.tsx

```diff
@@ -100,31 +100,31 @@ const CtPOneTimePasswordInput = (props: CtPOneTimePasswordInputProps): h.JSX.Ele
     }, [data, valid, errors]);
 
     return (
-        <Field
-            name="oneTimePassword"
-            label={i18n.get('ctp.otp.fieldLabel')}
-            labelEndAdornment={
```

Comment:
@ribeiroguilherme this was the reason the test was failing.

Basically we were creating the button inside the label, which made that the label invalid since it can only contain text. So all the selectors where not picking it up.

So I have refactored it a bit and moved the button to the outside of the label and used so CSS trickery to place the button where it was previously. 

My main concern with the change is the change in classnames, but let me know what you think.

Reply:
The change is fine - it was not following the correct approach and now it is, so there is no discussion about it. I am just concern about merchant's customization in this case. 

Some classnames changed, and merchants might have target the 'links' CSS and this impact their changes, but seems like there is not much to do in this case as we need to apply this fix.

Reply:
Yeah, that's what I was concerned too. I tried to minimize the impact of those changes, but I wanted to know your opinion on it.

In that case let's proceed with this.